recipes/rill/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rill" %}
+{% set version = "0.38.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rilldata/rill/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 01becbab8dc32f7c5c5fa05bce1e67827ea099edcd4e145a3137f79d8a6ffcc8
+  - url: https://gitlab.com/cznic/mathutil/-/raw/b13e5b5643328f15fd2fcedc85f647f0d8f9180f/LICENSE  # v1.5.0
+    sha256: bfa9bf72a72ca009fd62a8f84fca3dca67e51d93af96352723646599898b6cf5
+    fn: LICENSE-mathutil
+  - url: https://raw.githubusercontent.com/uber-go/zap/56468e797f1dbf3905ad7c34b5513928206244bf/LICENSE.txt  # v1.25.0
+    sha256: 7de716e70addb64f9305298ef32a9dd68e40d5b3095a5d868ba4461404dbfbcf
+    fn: LICENSE-zap
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('go-cgo') }}
+    - compilers
```

Comment:
Adding `compilers` as a build dependency is necessary, as otherwise you get the following error during the `go-licenses save` step.

```
$GOPATH/go.mod exists but should not
```

Adding the `compilers` package somehow unsets the `GOPATH` environment variable. However, it seems to be doing more than just that, as unsetting it without having `compilers` as a dependency did not work.

Reply:
The `compilers` packages should never ever be used inside a recipe. See below, you need a C++ compiler.